htdocs/PHP/internal_libraries/languages/page_content_fr.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+/**
+* Translations for page contents
+* Language: French
+*/
+
+$translation = array();
+
+// Title
+$translation["title"] = "WeExpire - Plus simple qu'un testament, plus sûr qu'une note.";
+
+// Description for search engines (max 160 characters)
+// The following string is exactly 160 chars
+$translation["meta_description"] = "WeExpire est un outil permettant de créer des notes d'urgence qui ne pourront être lues par vos contacts de confiance qu'après votre décès ou en cas de blessure.";
+
+// Description for social media (max 100 characters)
+// The following string is exactly 98 chars
+$translation["social_media_description"] = "Créez des notes d'urgence qui ne pourront être lues par vos contacts proches qu'après votre décès.";
+
+// Footer section
+$translation["footer_language"] = "Langue";
+$translation["footer_about"] = "À propos";
+$translation["footer_donate"] = "Dons";
+$translation["footer_contact"] = "Contact";
+$translation["footer_legal"] = "Légal";
+$translation["footer_privacy"] = "Vie privée";
+$translation["footer_terms"] = "Termes et conditions";
```

Comment:
It would be great if in French we can use just "Termes" instead of "Termes et conditions". In this way we create more space in the footer for mobile devices.